spinalcordtoolbox/scripts/sct_maths.py

```diff
@@ -375,7 +373,12 @@ def main(argv=None):
         data_out = sct_math.denoise_nlmeans(data, patch_radius=p, block_radius=b)
 
     elif arguments.symmetrize is not None:
```

Comment:
I think the convention we've been following for other `sct_maths` functions has been to create a function in `spinalcordtoolbox/math.py`, then import the function into `sct_maths`.

Reply:
yes absoluteltly!